master/internal/api_experiment.go

```diff
@@ -391,9 +391,13 @@ func (a *apiServer) GetExperiments(
 		ColumnExpr("(w.archived OR p.archived) AS parent_archived").
 		ColumnExpr("p.user_id AS project_owner_id").
 		Column("e.config").
+		ColumnExpr("e.group_id").
+		ColumnExpr("CASE WHEN e.group_id IS NULL THEN NULL ELSE g.name END AS group_name").
 		Join("JOIN users u ON e.owner_id = u.id").
 		Join("JOIN projects p ON e.project_id = p.id").
-		Join("JOIN workspaces w ON p.workspace_id = w.id")
+		Join("JOIN workspaces w ON p.workspace_id = w.id").
+		Join("LEFT JOIN experiment_groups g ON e.group_id IS NOT NULL " +
+			"AND e.group_id = g.id AND e.project_id = g.project_id")
```

Comment:
I believe that LEFT JOIN allows you to write the join without these checks for compatibility, for example I have no groups in my DB but I am able to run:

```sql
SELECT e.id, g.name FROM experiments e LEFT JOIN experiment_groups g ON e.group_id = g.id
```